js/background.js

```diff
@@ -359,22 +359,18 @@
             }
         }
 
-        // If we've just upgraded to read receipt support on desktop, kick off a
-        // one-time configuration sync request to get the read-receipt setting
-        // from the master device.
-        var readReceiptConfigurationSync = 'read-receipt-configuration-sync';
-        if (!storage.get(readReceiptConfigurationSync)) {
-
-            if (!firstRun && textsecure.storage.user.getDeviceId() != '1') {
-                textsecure.messaging.sendRequestConfigurationSyncMessage().then(function() {
-                    storage.put(readReceiptConfigurationSync, true);
-                }).catch(function(e) {
-                    console.log(e);
-                });
-            }
-        }
+    /* eslint-enable */
+    const deviceId = textsecure.storage.user.getDeviceId();
+    const { sendRequestConfigurationSyncMessage } = textsecure.messaging;
+    const status = await Signal.Startup.syncReadReceiptConfiguration({
+      deviceId,
+      sendRequestConfigurationSyncMessage,
+      storage,
+    });
+    console.log('Sync read receipt configuration status:', status);
```

Comment:
This will let us see the sync read receipt configuration status in every user’s log and help debugging.